### PR TITLE
fix: improve error message for git secrets duplication

### PR DESCRIPTION
**Pull request by mshaposhnik**
### What does this PR do?
Mre informative message in case of git credentials secret duplication in user namespace.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/GitCredentialStorageFileSecretApplier.java [hunk 1]
```diff
@@ -68,7 +69,10 @@ public void applySecret(KubernetesEnvironment env, RuntimeIdentity runtimeIdenti
       if (gitConfig != null) {
         if (gitConfig.contains("helper = store --file") && gitConfig.contains("[credential]")) {
           throw new InfrastructureException(
-              "Multiple git credentials secrets found. Please remove duplication.");
+              format(
+                  "Multiple git credential secrets for user %s found in namespace %s. That may be caused by reinstalling product without user namespaces cleanup or using multiple instances of product with the same namespace namings template.",
```
Comment (skabashnyuk) [inline infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/GitCredentialStorageFileSecretApplier.java:73]: can we add the exact names of the secrets?

Comment (mshaposhnik) [reply infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/GitCredentialStorageFileSecretApplier.java:73]: not easy cuz we processing all kinds of secrets in NS ant call appropriate applier with only given secret.

**Review by amisevsk — APPROVED** [review]

**Review by sparkoo — APPROVED** [review]

**Comment by sparkoo** [comment]
PR title is misleading. is it really "commit message" ?

**Review by skabashnyuk — APPROVED** [review]

**Comment by skabashnyuk** [comment]
Can you add a test for this case?

**Comment by l0rd** [comment]
@mshaposhnik is there an issue related to this PR?

**Comment by mshaposhnik** [comment]
@l0rd related to https://issues.redhat.com/browse/CRW-2013

**Pull request merged**